finance.reckoner.app.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: finance.reckoner.app
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+command: reckoner
+separate-locales: false
+finish-args:
+  [
+    --socket=x11,
+    --socket=wayland,
+    --share=network,
+    --filesystem=home,
+    --share=ipc,
+    --device=dri,
+  ]
+modules:
+  - name: reckoner
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin /app/reckoner /app/share/applications /app/share/icons/hicolor/192x192/apps
+      - cp finance.reckoner.app.png /app/share/icons/hicolor/192x192/apps/
+      - cp finance.reckoner.app.desktop /app/share/applications/
+      - cp -R bundle/* /app/reckoner/
+      - ln -s /app/reckoner/reckoner /app/bin/reckoner
+    sources:
+      - type: archive
+        only-arches: [x86_64]
+        dest: bundle
+        url: https://codeberg.org/Reckoner/app/releases/download/v1.0.2/reckoner-1.0.2-x86_64-linux.tar.gz
```

Comment:
how does this work on aarch64?

Reply:
This is currently only built for x86_64. Is there something else I need to specify to build just for x86?

Reply:
https://docs.flathub.org/docs/for-app-authors/maintenance#limiting-the-set-of-architectures-to-build-on